### ISSUE TITLE
What is the context about the example of pre-trained model usage?

### ISSUE THREAD

**Issue by ValerianRey** [OxWearables/ssl-wearables#10]
Hi! First of all, thanks for this high quality and open research!

I ran the usage example shown in the `README.md`:
```python
import torch
import numpy as np

repo = 'OxWearables/ssl-wearables'
harnet5 = torch.hub.load(repo, 'harnet5', class_num=5, pretrained=True)
x = np.random.rand(1, 3, 150)
x = torch.FloatTensor(x)
harnet5(x)

harnet10 = torch.hub.load(repo, 'harnet10', class_num=5, pretrained=True)
x = np.random.rand(1, 3, 300)
x = torch.FloatTensor(x)
harnet10(x)

harnet30 = torch.hub.load(repo, 'harnet30', class_num=5, pretrained=True)
x = np.random.rand(1, 3, 900)
x = torch.FloatTensor(x)
harnet30(x)
```

I have trouble understanding exactly what the harnet models are. I understand that they are instances of `sslearning.models.accNet.Resnet`, and my understanding is the following:
- The `feature_extractor` part was pre-trained using multi-task SSL on UK-Biobank dataset, as explained in [your paper](https://arxiv.org/pdf/2206.02909.pdf)
- The `classifier` part is not trained at all, and we have to train it on any labelled classification dataset (what makes me understand this is that we can select any number `class_num`).

Is my understanding correct?
Also, would it be possible to write this information next to the usage example?

Thanks a lot in advance :)

**Comment by angerhang** [comment]
@ValerianRey 

Thanks for the query. Appreciate your interest in our work.

Your understanding is correct:
* `feature_extractor` is the pre-trained network.
* `classifier` is just a task head that supports an arbitrary number of labels.

We are very busy with a few upcoming deadlines right now. If you wish to make a pull request, very happy to take a look and merge it in :D

**Comment by ValerianRey** [comment]
Thanks for your answer! I made a PR. Hopefully my explanation is accurate enough!
